tests/CMakeLists.txt

```diff
@@ -55,7 +56,7 @@ endif()
 target_link_libraries(jwt-cpp-test PRIVATE jwt-cpp nlohmann_json::nlohmann_json
                                            $<$<NOT:$<CXX_COMPILER_ID:MSVC>>:${CMAKE_DL_LIBS}>)
 
-gtest_add_tests(TARGET jwt-cpp-test)
+gtest_discover_tests(jwt-cpp-test)
```

Comment:
👏 https://cmake.org/cmake/help/latest/module/GoogleTest.html newer is better

and it's supported already https://github.com/Thalhammer/jwt-cpp/blob/6fed66d0d54911dd4a9eedd71abe745f7a3d9bd5/CMakeLists.txt#L1C37-L1C37